Makefile

```diff
@@ -204,25 +204,31 @@ docs: | build deps
 
 ## RLN
 
+compile_rln =
+
 # control rln code compilation
 ifeq ($(RLN), true)
-NIM_PARAMS := $(NIM_PARAMS) -d:rln
-else ifeq ($(CI), true)
-NIM_PARAMS := $(NIM_PARAMS) -d:rln
+compile_rln = true
 endif
 
 # control compilation of rln tests that require on chain interaction
 ifeq ($(ONCHAIN_RLN), true) 
+compile_rln = true
```

Comment:
What about putting everything under `compile_rln` and renaming it to `experimental_rln`?

This is what I was planning for the static compilation PR.

Reply:
Agreed, addressed in [055b387](https://github.com/waku-org/nwaku/pull/1410/commits/055b387a3d35467ad85fe50f7576a9af1683e2b9)